testing/itest/src/infra/mod.rs

```diff
@@ -300,6 +300,11 @@ impl SubnetInfra {
             jsonrpc_api_ws: None,
             auth_token: Some(admin_token),
             accounts,
+            evm: false,
+            eth_private_key: None,
+            chain_id: None,
```

Comment:
The chainID won't be needed, it can be infered from the subnetID. See: https://github.com/consensus-shipyard/ipc-actors/pull/109/files#diff-41548e2e7ed0210b54962a6cde729ac906ac66516f1c87615c4db169b3230930R71

Reply:
updated.